data/src/main/java/io/gnosis/data/adapters/MoshiBuilder.kt

```diff
@@ -14,8 +15,13 @@ internal val transactionDtoAdapter =
         .withDefaultValue(UnknownTransactionDto)
 
 val dataMoshi =
-    MoshiBuilderFactory.makeMoshiBuilder()
+    Moshi.Builder()
```

Comment:
Why can the factory not be used anymore? Or other question why is the order important, shouldn't the annotation make it order independent.

Reply:
we still use factory. But I think it is more feasible to have moshi constructed in the app itself and not in the library. Library should not make too many assumptions about how moshi will be used or what type adapters will be needed.

Reply:
Where do we use it? Shouldn't we use the same approach everywhere?

Reply:
we probably won't need that many adapters/type converters in other places. Also having a complete builder at one place is better for readability I think.

Reply:
This feels weird, if we think it is better to use then use it. In general I am more a fan of being explicit here and would remove the factory (as a lot of stuff in the moshi builder is order dependant)

Reply:
here is pr for this: https://github.com/gnosis/svalinn-kotlin/pull/49 
I would release new patch version for savlinn and update dependencies within this pr @rmeissner